.travis.yml

```diff
@@ -29,3 +32,5 @@ script:
   - npm run lint:js
   - npm run build --environment production
   - npm test
+  - npx ember build -prod
```

Comment:
Aren't we already building a prod build in couple of steps above?

Maybe in a future PR we could consider splitting these steps as different travis stages so that the build can be run in parallel to the lighthouse analysis.

Reply:
I'm probably just a little distracted this week that I didn't notice 🙈 we can delete the earlier one, we just need to make sure that the prod build is the step right before the lighthouse so it runs against the production optimised build

Sure we could do stages but I think it's probably fine to leave it like this for simplicity sake 🤔

Reply:
Fixed now 👍